submissions/bilkisuTace/phpDatabases/customers.php

```diff
@@ -0,0 +1,43 @@
+<?php
+include('db_con.php');
+include('setCookies.php');
+
+$sql = "SELECT customer_id, first_name, last_name, email, created_at FROM customers";
+$result = mysqli_query($conn, $sql);
```

Comment:
The idea of this task is to store the result returned from this SQL query in a cookie, Each time I reload this page, I don't want to always run this SQL query but instead, I want to get it from the data stored in the cookie as long as the cookie has not expired. If the cookie is has expired then run the SQL query again.

With your current implementation, each time I reload this page, I am always executing this SQL query.